packages/appcues-custom-previewer/android/build.gradle

```diff
@@ -58,6 +58,7 @@ android {
   defaultConfig {
     minSdkVersion getExtOrIntegerDefault("minSdkVersion")
     targetSdkVersion getExtOrIntegerDefault("targetSdkVersion")
+    buildConfigField("boolean", "IS_NEW_ARCHITECTURE_ENABLED", isNewArchitectureEnabled().toString())
```

Comment:
this `isNewArchitectureEnabled` comes from that flag from the [previous PR](https://github.com/appcues/mobile-pattern-showcase-app/pull/95/files#diff-96677aa35debfefd031d9d34d9c70369754ee3acb2d9a9d4090e98612efee6f5R13)

Reply:
Indirectly, yes. The expo config flag is propagated through the RN build process and read here.